.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,137 @@
+on:
+  # Branch deletion
+  delete:
+  pull_request:
```

Comment:
Do we need `delete` and `pull_request` here since this whole object is effectively:

```js
{
  "on": {
    "delete": null,
    "pull_request": null,
    "push": {
      "branches": [
        "master"
      ]
    }
  }
}
```

Reply:
I think I meant for it to run on all deletes so that it could catch branches being deleted, but it's only running (normally) on the master branch so it does make sense to remove `delete`. I wanted it to run on all pull_request actions, is this not the right syntax for that?